mod.conf

```diff
@@ -1,4 +1,4 @@
 name = drawers
 description = Storage Drawers - A Mod adding simple storages for items and blocks, showing the item's inventory image in the front.
-optional_depends = default, mcl_core, screwdriver, pipeworks, intllib, moreores, digilines, mesecons_mvps
+optional_depends = default, mcl_core, screwdriver, pipeworks, tubelib, intllib, moreores, digilines, mesecons_mvps
 min_minetest_version = 5.0
```

Comment:
> Requires Minetest 5.3.0 or newer.

Due to `minetest.is_creative_enabled` which was available since 5.3.0-dev

Reply:
Done